toplevel/coqloop.ml

```diff
@@ -371,8 +371,89 @@ let exit_on_error =
   declare_bool_option_and_ref ~depr:false ~name:"coqtop-exit-on-error" ~key:["Coqtop";"Exit";"On";"Error"]
     ~value:false
 
+(* XXX: extension of Vernacentries.show_proof *)
+let show_proof_diff_to_pp pstate =
```

Comment:
Why XXX?

Reply:
This seems like a TODO from the PR that introduced this code, indeed seems pretty messy to duplicate here so this should be improved.

Once I have a bit of time I'll resume #8728 , or anyone else is welcome to do it; I believe there is room to cleanup that code.

Reply:
It is not a TODO, it explains what the code does.
It may be an irrelevant comment, but how can it be a todo?

Reply:
I read it as "this is duplicate code, show_proof should be amended" but indeed I guess it depends on how interpret the original intention.

Reply:
Updated the comment, let me know how it looks know.